espnet/nets/pytorch_backend/transformer/subsampling.py

```diff
@@ -41,10 +39,13 @@ def forward(self, x, x_mask):
         :param torch.Tensor x_mask: input mask
         :return: subsampled x and mask
         :rtype Tuple[torch.Tensor, torch.Tensor]
+               or Tuple[Tuple[torch.Tensor, torch.Tensor], torch.Tensor]
         """
         x = x.unsqueeze(1)  # (b, c, t, f)
         x = self.conv(x)
         b, c, t, f = x.size()
+        # if RelPositionalEncoding, x: Tuple[torch.Tensor, torch.Tensor]
```

Comment:
Can you remove unrelated comments or if you leave these lines please explain more why you put these comments.

Reply:
Ok, I will remove these comments.